scripts/conversion_toolkits/convert_t5.py

```diff
@@ -0,0 +1,100 @@
+
+
+def parse_args(): 
+    pass
```

Comment:
Let's try to add the argparse and add the huggingface model name as an argument, e.g., t5-small, t5-base, t5-large.